renderer/src/components/WalletModule.tsx

```diff
@@ -64,7 +64,7 @@ const WalletModule: FC<PropsWallet> = ({ isOpen = false }) => {
     <div className='relative z-20'>
       <div className='h-8 bg-primary-dark flex items-center px-8' onClick={reset}>
         <p className='text text-body-3xs text-white opacity-80 mr-3'>STATION ADDRESS</p>
-        <p className='text text-body-3xs text-white'>{stationAddress}</p>
+        <p className='text text-body-3xs text-white'>{`${stationAddress.slice(0, 6)} ... ${stationAddress.slice(-6)}`}</p>
```

Comment:
Let's use the ellipsis character ('…', U+2026 HORIZONTAL ELLIPSIS) instead of `...`.

See e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow and https://www.htmlsymbol.com/html-entities/horizontal-ellipsis.html

```suggestion
        <p className='text text-body-3xs text-white'>{`${stationAddress.slice(0, 6)} &hellip; ${stationAddress.slice(-6)}`}</p>
```

Reply:
great idea, done!